### PR TITLE
163361831 compare days 500 error

### PR DESCRIPTION
**Pull request by rckopteri**
# Fixed
* Changed gtfs_route_difference stored procedure to work with route-hash-id only.
* This prevents finding multiple route differences and fixes the bug.
* Also added few random numbers to table component to prevent js console warnings. However our table component doesn't take into account :keys that are passed to header row it is always 'h1' and it cannot be fixed using this version of material ui.

### CODE REVIEW

File: ote/src/cljs/ote/ui/table.cljs [hunk 1]
```diff
@@ -19,6 +19,7 @@
                      label-style
                      row-style show-row-hover?
                      on-select row-selected? no-rows-message class] :as opts} headers rows]
+  (let [random-table-id (str "tid-"(rand-int 999))]
```
Comment (jkorpelin) [inline ote/src/cljs/ote/ui/table.cljs]: Maybe (java.util.UUID/randomUUID) instead of rand-int? Lower chance of collision.

Comment (philosotractor) [reply ote/src/cljs/ote/ui/table.cljs]: Or id via `range` to have no chance of collision?

Comment (philosotractor) [reply ote/src/cljs/ote/ui/table.cljs]: For the sake of discussion, another alternative is to pass the table :key value from view who is creating the table, to here. View could assign unique keys using a string and an iterated integer.

File: database/src/main/resources/db/migration/R__GTFS_query.sql [hunk 2]
```diff
@@ -392,9 +391,9 @@ BEGIN
 
   --RAISE NOTICE 'yli jäi d1: % ja d2: % vuoroa', array_length(d1_trip_ids, 1), array_length(d2_trip_ids, 2);
 
-  route_changes."route-short-name" := "route-short-name";
-  route_changes."route-long-name" := "route-long-name";
-  route_changes."trip-headsign" := "trip-headsign";
+  route_changes."route-short-name" := null; --"route-short-name";
```
Comment (philosotractor) [inline database/src/main/resources/db/migration/R__GTFS_query.sql]: Are the old values left in comments on purpose?

Comment (rckopteri) [reply database/src/main/resources/db/migration/R__GTFS_query.sql]: route_changes uses composition type which requires something to be added on those columns.
When we migrate to composite type to own table, this won't be needed anymore. So yes and no :) we don't need those comments but this is going to change soon anyway.

File: ote/src/cljs/ote/ui/table.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/ui/table.cljs]: I leave it as it is. We have max 3 tables on same page so there is very low change of collision. We have a ticket for creating new table component and I don't want to spend any more time fixing this old one.

**Pull request merged**